### PR TITLE
Install Veeam Agent for Microsoft Windows documentation

### PR DESCRIPTION
**Pull request by Viaceslav-Kodesh**

### CODE REVIEW

File: _docs/tutorials/win-veeam-agent.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+> Create a recovery image for your computer.
+
+## Upgrading Veeam Agent for Windows
+You can start the upgrade process from the Veeam Agent for Windows Control Panel when the new version becomes available.
+
+During the upgrade process, configuration and backup files that were created with the previous version of Veeam Agent for Windows are not impacted in any way.
+
+## To upgrade Veeam Agent for Windows
+Open the Veeam Agent for Windows icon in the system tray, or right-click the Veeam Agent for Windows icon in the system tray and select control Panel and open the About tab.
```
Comment (deniskodesh) [inline _docs/tutorials/win-veeam-agent.md]: Control Panel

File: _docs/tutorials/win-veeam-agent.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+title: Install Veeam Agent for Microsoft Windows
+description: How to install Veeam Agent for Microsoft Windows
+tags: [ featured, coretasks ]
+# permalink: /install-Veeam-Agent-for-Microsoft-Windows/
+---
+# Install Veeam Agent for Microsoft Windows
+{: .no_toc }
+---
+
+In this page, you can find an explanation of how to Install and use Veeam Agent for Microsoft Windows.
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:11]: install

File: _docs/tutorials/win-veeam-agent.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Upgrading Veeam Agent for Windows
+You can start the upgrade process from the Veeam Agent for Windows Control Panel when the new version becomes available.
+
+During the upgrade process, configuration and backup files that were created with the previous version of Veeam Agent for Windows are not impacted in any way.
+
+## To upgrade Veeam Agent for Windows
+Open the Veeam Agent for Windows icon in the system tray, or right-click the Veeam Agent for Windows icon in the system tray and select Control Panel and open the About tab.
+If the new version of Veeam Agent for Windows is available, **click Download**.
+When the download is complete, **click Install**.
+To upgrade Veeam Agent for Windows, you must accept the license agreement. Read the license agreement, select the I agree to the Veeam End User License Agreement check box and click Update.
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:62]: select the *I agree to the Veeam End User License Agreement* check box and click  **Update**.

File: _docs/tutorials/win-veeam-agent.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Consider the following
+In some cases, upgrade to the new version of Veeam Agent for Windows may require computer reboot.
+Save the downloaded archive on the computer where you plan to install the new version of the product and double-click the setup archive to start the upgrade.
+After you upgrade Veeam Agent for Windows, you must obtain and install a new license.
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:67]: After upgrading Veeam Agent for Windows, you must obtain and install a new license.

File: _docs/tutorials/win-veeam-agent.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## Uninstalling Veeam Agent for Windows
+From the Start menu, select Control Panel > Programs and Features.
+In the programs list, right-click Veeam Agent for Windows and select Uninstall. Wait for the process to complete.
+The Local DB and other prerequisite components installed and used by Veeam Agent for Windows are not removed during the uninstall process. To remove each of the remaining components, right-click it in the programs list and select Uninstall.
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:72]: The Local DB and other required components installed and used by Veeam Agent for Windows are not removed during the uninstallation process.

File: _docs/tutorials/win-veeam-agent.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+In this page, you can find an explanation of how to Install and use Veeam Agent for Microsoft Windows.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+# Before You Begin
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:19]: maybe it's an unnecessary title

File: _docs/tutorials/win-veeam-agent.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+> Microsoft SQL Server 2012 Management Objects
+
+If these components are not pre-installed on the computer, the setup will install them automatically.
+
+The product program files are placed to the **%ProgramFiles%\Veeam\Endpoint** Backup folder on the system volume. Veeam Agent for Windows requires at least **200 MB**.
+During the recovery image creation, Veeam Agent for Windows formats the removable storage device.
+
+## To install Veeam Agent for Windows
+Save the downloaded archive on the computer where you plan to install the product.
+You should accept the license agreements and click Install.
```
Comment (Vika1990Z) [inline _docs/tutorials/win-veeam-agent.md:41]: you may need to add text selection: 
and click **Install**.

File: _docs/tutorials/win-veeam-agent.md (hunk 2 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:11]: Fixed

File: _docs/tutorials/win-veeam-agent.md (hunk 3 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:62]: Fixed

File: _docs/tutorials/win-veeam-agent.md (hunk 4 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:67]: With this fix the meaning has not changed.

File: _docs/tutorials/win-veeam-agent.md (hunk 5 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:72]: Fixed

File: _docs/tutorials/win-veeam-agent.md (hunk 6 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:19]: Sure

File: _docs/tutorials/win-veeam-agent.md (hunk 7 above)
Comment (Viaceslav-Kodesh) [reply _docs/tutorials/win-veeam-agent.md:41]: Fixed

**Pull request merged**